### ISSUE TITLE
[Bug] Inconsistent review count

### ISSUE THREAD

**Issue by lervag** [ankidroid/Anki-Android#11710]
###### Reproduction Steps

This has started happening sometime in the last week or two.

1. I start my Anki session and notice the review count on the "main screen" (list of decks) when I start AnkiDroid. I have only a single deck ("Main").
2. I start reviewing some cards, then I return to the list of decks.
3. The review count suddenly drops _more_ than the number of cards I reviewed.

I did not find any other issues on this topic. I'll be glad to submit more information. For example, I could take screenshots tomorrow and highlight the problem. I'm using the beta version (but can't find the specific version I'm on right now).

**Comment by mikehardy** [comment]
More than anything it would be nice to have the information requested in the template for new issues, specifically the debug info :smile: 

https://github.com/ankidroid/Anki-Android/blob/main/.github/ISSUE_TEMPLATE/issue_template.md#debug-info

Also, if you can reproduce this on new releases or not https://github.com/ankidroid/Anki-Android/releases

**Comment by mikehardy** [comment]
yeah we are blocked on releases right now (alpha or otherwise) until conclusion of #5304 - just download the appropriate APK direct and you'll be updated, only 40 versions behind ;-) :-) https://github.com/ankidroid/Anki-Android/releases

**Comment by lervag** [comment]
Ah, that explains why I've not had any updates. I never noticed, AnkiDroid has generally just worked as expected in the meantime :)

I'll try and update and see if this fixes my issue. I'll let you know tomorrow!

**Comment by lervag** [comment]
By the way, are there any summaries for the AnkiDroid changelog? It would be interesting to read up on the changes since November last year. Of course, I could browse the commit log :)

**Comment by lervag** [comment]
Impressive! I'm very much looking forward to the release and reading the release notes on 2.16! :D

**Comment by david-allison** [comment]
A quick highlight of a few features (copied from a post I made a few months back):

* Allow users to draw new cards using the whiteboard
* TTS API (usable with AnKing cards)
* Improved Note Editor toolbar
* Video Clip Support
* Auto night mode
* More gestures + better gesture support
* Gamepad/Remote/Keyboard Support
* Improved Deck Downloads
* Improved Preferences

**Comment by lervag** [comment]
Very cool!

But ok, back to the present issue: I'm still observing the inconsistent due count. Here's a side-by-side screenshot before and after opening the Main deck:

![out](https://user-images.githubusercontent.com/998898/176407447-9913ea49-d38c-4932-9c97-0fe5deed23b8.jpg)

I don't know how to debug this issue, so any help is appreciated.

**Comment by mikehardy** [comment]
For what it's worth, the latest 2.16alpha has an advanced setting addition that lets you turn on the "new backend" (basically: an updated version of the Anki Desktop Rust code used directly here) as well as the v3 scheduler, if you'd like to help test it. Could fix this issue too, if it's still happening https://github.com/ankidroid/Anki-Android/releases

Instructions to test https://github.com/ankidroid/Anki-Android/issues/10411#issuecomment-1224591312

**Comment by lervag** [comment]
Thanks! I've updated to the latest alpha and enabled the v3 scheduler. I've first synced everything to my computer and took a backup of my files there.
